configure_logging.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
-set -v
+set -euv
```

Comment:
Exit on first error since we don't want to continue with a partially configured system.  Do not tolerate unset variables because it is dangerous.